src/pages/guidelines/color/overview.mdx

```diff
@@ -250,7 +250,10 @@ In the dark themes, layers become one step lighter with each added layer.
 
 <Tab label="Gray 90">
 
-![A dashboard in the Gray 90 theme](images/color-overview-themes-gray90.png)
+Gray 90 components are paired with Gray 100 backgrounds to make the Gray 100
```

Comment:
This is the old content being imported here. We need to keep the image instead.

Reply:
updated!